src/icons/src/lightning.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="43.998" height="67" viewBox="0 0 43.998 67">
```

Comment:
Deze is wel erg groot in vergelijking met andere iconen. Ik denk dat de anderen allemaal 30x30 px zijn. Het lijkt me nuttig om de sizes gelijk te houden zodat de var(--icon-size-sml) values dan overal hetzelfde willen zeggen (een veelvoud van die 30px). Kan je de size van de andere iconen eens bekijken, deze aanpassen in Sketch, exporteren, en de icoontjes opnieuw genereren?